src/views/ShareCredentialSign/ShareCredentialSign.tsx

```diff
@@ -135,7 +135,16 @@ export function ShareCredentialSign({
             <div key={sharedProp} className={styles.detail}>
               <dt className={styles.detailName}>{sharedProp}</dt>
               <dd className={styles.detailValue}>
-                {credential.request.claim.contents[sharedProp]}
+                {/* TODO: How to display object values? */}
```

Comment:
Do we have a ticket or something to actually address all these TODOs? I am concerned that they’re going to just stay here forever.

Reply:
I'll create a ticket

Reply:
If the ticket is new, we’ll also have to push for this question to be answered quickly

Reply:
I personally would prefer to just have it as `String(value)`, as we don’t support nested values yet